galleon-pack/src/main/resources/feature_groups/distributable-web.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature-group-spec name="distributable-web" xmlns="urn:jboss:galleon:feature-group:1.0">
+
+    <feature spec="subsystem.distributable-web">
+        <param name="default-session-management" value="default"/>
+        <param name="default-single-sign-on-management" value="default"/>
+        <feature spec="subsystem.distributable-web.infinispan-session-management">
+            <param name="infinispan-session-management" value="default"/>
+            <param name="cache-container" value="web"/>
+            <param name="granularity" value="SESSION"/>
+            <feature spec="subsystem.distributable-web.infinispan-session-management.affinity.primary-owner"/>
+        </feature>
+        <feature spec="subsystem.distributable-web.infinispan-single-sign-on-management">
+            <param name="infinispan-single-sign-on-management" value="default"/>
+            <param name="cache-container" value="web"/>
+            <param name="cache" value="sso"/>
+        </feature>
+        <feature spec="subsystem.distributable-web.routing.infinispan">
```

Comment:
A non-clustered version of this feature-spec-group would want to use:
<feature spec="subsystem.distributable-web.routing.local"/>